docs-src/concepts/what-is-cluster-configuration.md

```diff
@@ -0,0 +1,127 @@
+---
+id: what-is-cluster-configuration
+title: What is Cluster configuration?
+sidebar_label: Cluster configuration
+description: Cluster Configuration is the setup and configuration details of your Temporal Cluster, defined using YAML.
+tags:
+  - term
+  - explanation
+---
+
+Cluster configuration is the setup and configuration details of your self-hosted Temporal Cluster, defined using YAML.
+You must define your Cluster configuration when setting up your Temporal Cluster.
+
+Cluster configuration is composed of two types of configurations: Static configuration and Dynamic configuration.
+
+### Static configuration
+
+Static configuration contains details of how the Cluster should be set up. The static configuration is read just once and used to configure service nodes container/pod at startup.
+Depending on how you want to deploy your self-hosted Temporal Cluster, your static configuration must contain details for setting up:
+
+- Temporal Services - Frontend, History, Matching, Worker
+- Membership ports for the Temporal Services
+- Persistence (including Shard count), Visibility and Advanced Visibility, Archival store setups.
+- TLS, Auth
+- Server log level
+- Metrics
+- Cluster Metadata
+- Dynamic Config Client
+
+Static configuration values cannot be changed at runtime.
+Some values, such as Metrics configuration or Server log level can be changed in the static configuration but require restarting the Cluster for the changes to take effect.
```

Comment:
does this mean user can only change prom port when restarting?

Reply:
It means that if you want to change the prometheus port on the any of the pods, you will need to restart your server for the update to apply. I don't think you can change the prom port without restarting or redeploying the cluster image; is this correct?